languages/go/exercises/concept/maps/.docs/after.md

```diff
@@ -0,0 +1,58 @@
+Further Reading:
+
+- [godoc-race-detector]
+- [goblog-race-detector]
+- [goblog-iteration-order]
+
+A `map` in Go is an associative data type that contains a collection of key/value pairs, which other languages might refer to as as dictionary or an associative array.
+
+The value of an uninitialized map is `nil`, and you can define a map as follows:
+
+```go
+make(map[string]int)
+```
+
+or
+
+```go
+  var foo map[string]int
+  foo = make(map[string]int)
+```
+
+> Here `string` represent the key type of map, and `int` represent the element type of map. You can change them to whatever valid golang type.
+
+To store a value in a map, you can use `=` operator like
+
+```go
+  foo["bar"] = 42
+```
+
+`map` is indexed by its key, therefore, you are required to use different key if you want to store a different data.
```

Comment:
```suggestion
`map` is indexed by its key, therefore, you are required to use a different key if you want to store different data.
```

Reply:
I'm wondering if this bit of text should be placed before the `=` assignment example before this.